ambry-clustermap/src/main/java/com/github/ambry/clustermap/AmbryPartition.java

```diff
@@ -42,7 +42,7 @@
   private long lastUpdatedSealedStateChangeCounter = 0;
 
   private static final short VERSION_FIELD_SIZE_IN_BYTES = 2;
-  private static final short CURRENT_VERSION = 1;
+  public static final short CURRENT_VERSION = 1;
```

Comment:
I think this could be package private now

Reply:
fixed.